src/handlers/actions/editDomain.ts

```diff
@@ -0,0 +1,36 @@
+import { utils } from 'ethers';
+import provider from '~/provider';
+import {
+  ColonyActionType,
+  ContractEvent,
+  ContractEventsSignatures,
+} from '~/types';
+import {
+  toNumber,
+  verbose,
+  writeActionFromEvent,
+  getDomainDatabaseId,
+} from '~/utils';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const receipt = await provider.getTransactionReceipt(event.transactionHash);
+  const hasDomainAddedEvent = receipt.logs.some((log) =>
+    log.topics.includes(utils.id(ContractEventsSignatures.DomainAdded)),
+  );
```

Comment:
Cool use of `.id` to skip having to parse the log to get the event's name.

It makes more sense like since you only need to check for existence